src/seml/utils/__init__.py

```diff
@@ -430,9 +431,12 @@ def __init__(
         name: str,
         time_to_live: float,
     ):
-        import seml.utils.typer as typer
+        from seml.settings import SETTINGS
 
-        self.cache_path = Path(typer.get_app_dir('seml')) / f'{name}.json'
+        user = os.environ.get('USER', 'unknown')
+        install_hash = hashlib.md5(name.encode('utf-8')).hexdigest()
+        file_name = f'seml_{user}_{name}_{install_hash}.json'
+        self.cache_path = Path(SETTINGS.TMP_DIRECTORY) / file_name
```

Comment:
Is this the most sensible way for this cache? Upon a superficial browse I also didn't find anything that is more reliable. Ideally, the tempfile also disappears automatically (e.g. by putting it into /tmp).

Reply:
SETTINGS.TMP_DIRECTORY is /tmp